auth/migrations/0000_init.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS users (
+  account_name TEXT PRIMARY KEY,
+  secret TEXT UNIQUE,
+  super_user BOOLEAN DEFAULT FALSE,
+  account_tier TEXT DEFAULT "basic" NOT NULL
```

Comment:
I'm thinking we can merge these two and only have `account_tier`?

Reply:
Sure, that makes sense.